examples/ring_app/ring-example.conf

```diff
@@ -30,6 +30,19 @@ authorization: {
             allow: "$1"
             name: "users allowed by backreference"
             sort-order: 500
-        }
+        },
+        {
+            match-request: {
+                path: "/hello/extension-allowed"
+                type: "path"
+            }
+            allow: {
+                extensions: {
+                    my_extension_shortname: "extensions value"
```

Comment:
Not a big deal either way, but I find it useful with these examples to have some ready-made certificates in the repo that I can play around with against the rules in the ring example.  For my testing, I just used the existing cert in the repo and experimented with two different allow/deny rules for the 'netscape comment' (2.16.840.1.113730.1.13) extension, since that extension already existed in the dummy cert in the repo.  Not worth adding anything else at this point, I think, but maybe something to keep in mind if we're making changes around this again in the future?